morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Is this intended to be a boolean value? If so it needs to be documented, but it seems like it would be really nice to move away from functions that have boolean arguments. They're a code smell.